content/guides/asset-management.md

```diff
@@ -126,41 +382,126 @@ npm install --save-dev csv-loader xml-loader
 
 __webpack.config.js__
 
-``` js
-module.exports = {
-  //...
-  module: {
-    rules: [
-      {
-        test: /\.(csv|tsv)$/,
-        use: 'csv-loader'
-      },
-      {
-        test: /\.xml$/,
-        use: 'xml-loader',
-      }
-    ]
-  },
-  //...
-}
+``` diff
+  var path = require('path');
+
+  module.exports = {
+    entry: './src/index.js',
+    output: {
+      filename: 'bundle.js',
+      path: path.resolve(__dirname, 'dist')
+    },
+    module: {
+      rules: [
+        {
+          test: /\.css$/,
+          use: [
+            'style-loader',
+            'css-loader'
+          ]
+        },
+        {
+          test: /\.(png|svg|jpg|gif)$/,
+          use: [
+            'file-loader'
+          ]
+        },
+        {
+          test: /\.(woff|woff2|eot|ttf|otf)$/,
+          use: [
+            'file-loader'
+          ]
+        },
++       {
++         test: /\.(csv|tsv)$/,
++         use: [
++           'csv-loader'
++         ]
++       },
++       {
++         test: /\.xml$/,
++         use: [
++           'xml-loader'
++         ]
++       }
+      ]
+    }
+  };
+
```

Comment:
I would drop this trailing line -- maybe we can add something to our linter to flag on these.